src/test/sitl/sitl_harness.py

```
@@ -0,0 +1,440 @@
+        self.sock = None
+        self.msp = None
+        self.boxids = []
+
+    def provision(self, cli_lines):
+        cfg = os.path.join(self.workdir, "scenario_config.txt")
+        with open(cfg, "w") as f:
+            f.write("\n".join(cli_lines) + "\n")
+        eeprom = os.path.join(self.workdir, "eeprom.bin")
+        if os.path.exists(eeprom):
+            os.remove(eeprom)
+        res = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)